snuba/datasets/metrics_bucket_processor.py

```diff
@@ -16,8 +18,10 @@
     ProcessedMessage,
     _ensure_valid_date,
 )
+from snuba.utils.streams.topics import Topic
 
 DISABLED_MATERIALIZATION_VERSION = 1
+METRICS_TOPIC = Topic.METRICS.value
```

Comment:
Hard coding the topic here since all messages processed by these processors will come from the metrics topic. This topic is just telling the DLQ (and in turn whoever looks at the bad messages in the dead letter topic) where the message was originally produced to. No implications on any logic.